### DISCUSSION TITLE
Add a "Show usages" button to all records in the form meta menu

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5879] [Feature Requests]
<img width="891" alt="image" src="https://github.com/specify/specify7/assets/37256050/c607c3b2-09d7-4a68-bfef-c1945ab5ae25">

It should function just like the delete blocker, except it doesn't make the user click the delete button.
Very useful for finding out how many records will be affected by a change (localities, collecting events, etc.)

**Comment by maxpatiiuk** [comment]
Can be added as a "Show usages" button in form meta (form meta already has a "Show usages" button when opened for a tree record or picklist record)

**Comment by grantfitzsimmons** [comment]
> One issue I noticed with Specify is that it does not tell you if a CE is shared amongst COs when viewing the CE form. Use of CEs can get complex because sometimes you want multiple COs to share the same one and other times you want unique ones.
> 
> Therefore an editor may add a specific note to the CE in the future with a specific CO in mind and not realize that all other CO's will point to this CE edit.
> 
> The only way currently to see if a CE is shared is to query the COs and CEs to determine this. I will submit this note to Specify.

**Reported by:** Matthew Cruz

**Comment by grantfitzsimmons** [comment]
Discussed today on https://discourse.specifysoftware.org/t/find-usages-feature-for-locality-records/1715/8

**Comment by danb213** [comment]
CSIRO would also like to see this feature, especially for agents as well as collecting events and localities. Just like the option in the "Merge records" form but on the agent forms: 
![image](https://github.com/specify/specify7/assets/110948439/933242d0-a67a-4317-b650-145c49a5acf4)
